keps/sig-node/20190412-lxcfs.md

```diff
@@ -0,0 +1,189 @@
+---
+kep-number: 
+title: kubelet support lxcfs
+authors:
+  - "@fanux"
+owning-sig: sig-node
+participating-sigs:
+reviewers:
+  - "@resouer"
+approvers:
+editor: fanux
+creation-date: 2019-04-12
+last-updated: 2019-04-12
+status: implemented
+see-also: 
+replaces: 
+superseded-by:
+---
+
+# Kubelet support lxcfs
+
+## Table of Contents
+
+
+   * [Kubelet support lxcfs](#lxcfs)
+      * [Table of Contents](#table-of-contents)
+      * [Summary](#summary)
+      * [Motivation](#motivation)
+         * [Goals](#goals)
+         * [Non-Goals](#non-goals)
+      * [Proposal](#proposal)
+         * [User Stories [optional]](#user-stories-optional)
+         * [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+         * [Risks and Mitigations](#risks-and-mitigations)
+      * [Graduation Criteria](#graduation-criteria)
+      * [Implementation History](#implementation-history)
+         * [Version 1.14](#version-114)
+         * [Version 1.15](#version-115)
+
+Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
+
+## Summary
+
+A proposal to enable pod container resource view, we get resource info in pod
+is the host resource when using resource quota, with lxcfs we can will have a
+right reource view in pod.
+
+## Motivation
+
+Java apps always abnormal run in pods with resource quota, because JVM allocate
```

Comment:
Would this JDK improvement handle your use case? https://bugs.openjdk.java.net/browse/JDK-8146115

Reply:
Yes, it can solve this problem, but the platform can't control user behavior